src/OpenLoco/src/Ui/Windows/Options.cpp

```diff
@@ -1679,7 +1679,7 @@ namespace OpenLoco::Ui::Windows::Options
             Dropdown::show(w->x + dropdown.left, w->y + dropdown.top, dropdown.width() - 4, dropdown.height(), w->getColour(WindowColour::secondary), _112C185, 0x80);
 
             int index = -1;
-            for (auto object : ObjectManager::getAvailableObjects(ObjectType::currency))
+            for (auto& object : ObjectManager::getAvailableObjects(ObjectType::currency))
```

Comment:
visual studio actually picks up on a couple of these "unintended copy" issues, I'll see if I can make a PR fixing them

Reply:
Argh this one wasn't meant to be in this pr